### ISSUE TITLE
SpongeCommon/SpongeForge command changes

### ISSUE THREAD

**Issue by Grinch** [SpongePowered/SpongeDocs#618]
Just wanted to note a few changes to the `/sponge plugins` command.

As of now SpongeCommon uses pagination for the plugin command to display installed plugins. On SpongeCommon/SpongeVanilla it forces `Minecraft, MCP, SpongeAPI, and Sponge` at the beginning of the list in that order. The console will display a non-paginated list for this command.

SpongeForge filters out mods from the `/sponge plugins` command and added a new command `/sponge mods` which will display all mods. This forces `Minecraft, MCP, FML, MinecraftForge, SpongeAPI, and Sponge` at the beginning of the list in that order. The console also displays a non-paginated list for this command. This command uses the permission `sponge.command.mods`.

Both commands also make use of tooltips for each listed item that allow you to hover over the plugin/mod and see the ID and Version quickly. Clicking it still acts the same by showing a more detailed output of the plugin/mod's properties.

**tl;dr**: `/sponge plugins` now is a paginated list when sent to players. Filters out mods on SpongeForge. SpongeForge added `/sponge mods` which displays installed mods and uses the permission `sponge.command.mods`

**Comment by Inscrutable** [comment]
Which versions of SpongeAPI does this affect (ie. is it being cherry-picked to older versions)?

**Comment by Grinch** [comment]
The only version I targeted for this is 7. It could be pulled into older versions but that isn't something I personally plan to PR.

**Comment by parlough** [comment]
I'll be working on an API 7 cleanup soon. Summer break here I come.

**Comment by ryantheleach** [comment]
In addition:

https://docs.spongepowered.org/stable/en/server/spongineer/commands.html#sponge

Needs to be double checked against https://github.com/SpongePowered/SpongeCommon/blob/stable-7/src/main/java/org/spongepowered/common/command/SpongeCommandFactory.java#L175

The blockinfo and entityinfo are missing, maybe others?

**Comment by ST-DDT** [comment]
@parlough Status?

**Comment by ST-DDT** [comment]
@parlough Do you mind if I take over?

**Comment by ST-DDT** [comment]
Note to self: The following commands are not or wrongly listed:

**Missing**

* /sponge blockinfo
* /sponge entityinfo
* /sponge which
* /callback

**Changed**

* /sponge help -> /help

**Comment by Inscrutable** [comment]
The `callback` command is exclusively used by plugins, so that's probably all that needs to be said about it.
